### PR TITLE
Fix path issues and add installation instructions

### PR DESCRIPTION
**Pull request by csfreak**

### CODE REVIEW

**Comment by csfreak** [comment]
Test of install via pip
```
jaross@jaross1-mac ➜  testparse  mkv
Created venv in '/Users/jaross/tmp/testparse/.venv'                                                                                                                                                                                  /1.4s
jaross@jaross1-mac ➜  testparse  activate
Activated virtual environment .venv                                                                                                                                                                                                  /0.0s
jaross@jaross1-mac ➜  testparse  venv:(.venv) pip install git+https://github.com/rhtools/vminfo-parser.git@feature/packaging
Collecting git+https://github.com/rhtools/vminfo-parser.git@feature/packaging
  Cloning https://github.com/rhtools/vminfo-parser.git (to revision feature/packaging) to /private/var/folders/19/3zqrltys2d1byd2f900qr40w0000gn/T/pip-req-build-5kytpxss
  Running command git clone --filter=blob:none --quiet https://github.com/rhtools/vminfo-parser.git /private/var/folders/19/3zqrltys2d1byd2f900qr40w0000gn/T/pip-req-build-5kytpxss
  Running command git checkout -b feature/packaging --track origin/feature/packaging
  Switched to a new branch 'feature/packaging'
  branch 'feature/packaging' set up to track 'origin/feature/packaging'.
  Resolved https://github.com/rhtools/vminfo-parser.git to commit f613511b8432c4d13ba21521e23167d17b3e3108
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting contourpy>=1.2.1 (from vminfo-parser==0.1.0.dev48)
  Using cached contourpy-1.3.0-cp312-cp312-macosx_11_0_arm64.whl.metadata (5.4 kB)
Collecting cycler>=0.12.1 (from vminfo-parser==0.1.0.dev48)
  Using cached cycler-0.12.1-py3-none-any.whl.metadata (3.8 kB)
Collecting et-xmlfile>=1.1.0 (from vminfo-parser==0.1.0.dev48)
  Using cached et_xmlfile-1.1.0-py3-none-any.whl.metadata (1.8 kB)
Collecting fonttools>=4.53.1 (from vminfo-parser==0.1.0.dev48)
  Using cached fonttools-4.53.1-cp312-cp312-macosx_11_0_arm64.whl.metadata (162 kB)
Collecting kiwisolver>=1.4.5 (from vminfo-parser==0.1.0.dev48)
  Using cached kiwisolver-1.4.7-cp312-cp312-macosx_11_0_arm64.whl.metadata (6.3 kB)
Collecting matplotlib>=3.9.1.post1 (from vminfo-parser==0.1.0.dev48)
  Using cached matplotlib-3.9.2-cp312-cp312-macosx_11_0_arm64.whl.metadata (11 kB)
Collecting numpy>=2.0.1 (from vminfo-parser==0.1.0.dev48)
  Using cached numpy-2.1.1-cp312-cp312-macosx_14_0_arm64.whl.metadata (60 kB)
Collecting openpyxl>=3.1.5 (from vminfo-parser==0.1.0.dev48)
  Using cached openpyxl-3.1.5-py2.py3-none-any.whl.metadata (2.5 kB)
Collecting packaging>=24.1 (from vminfo-parser==0.1.0.dev48)
  Using cached packaging-24.1-py3-none-any.whl.metadata (3.2 kB)
Collecting pandas>=2.2.2 (from vminfo-parser==0.1.0.dev48)
  Using cached pandas-2.2.2-cp312-cp312-macosx_11_0_arm64.whl.metadata (19 kB)
Collecting pillow>=10.4.0 (from vminfo-parser==0.1.0.dev48)
  Using cached pillow-10.4.0-cp312-cp312-macosx_11_0_arm64.whl.metadata (9.2 kB)
Collecting pyparsing>=3.1.2 (from vminfo-parser==0.1.0.dev48)
  Using cached pyparsing-3.1.4-py3-none-any.whl.metadata (5.1 kB)
Collecting PyQt5>=5.15.11 (from vminfo-parser==0.1.0.dev48)
  Using cached PyQt5-5.15.11-cp38-abi3-macosx_11_0_arm64.whl.metadata (2.1 kB)
Collecting PyQt5-Qt5>=5.15.14 (from vminfo-parser==0.1.0.dev48)
  Downloading PyQt5_Qt5-5.15.15-py3-none-macosx_11_0_arm64.whl.metadata (536 bytes)
Collecting PyQt5-sip>=12.15.0 (from vminfo-parser==0.1.0.dev48)
  Using cached PyQt5_sip-12.15.0-cp312-cp312-macosx_10_9_universal2.whl.metadata (421 bytes)
Collecting python-dateutil>=2.9.0.post0 (from vminfo-parser==0.1.0.dev48)
  Using cached python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata (8.4 kB)
Collecting python-magic>=0.4.27 (from vminfo-parser==0.1.0.dev48)
  Using cached python_magic-0.4.27-py2.py3-none-any.whl.metadata (5.8 kB)
Collecting pytz>=2024.1 (from vminfo-parser==0.1.0.dev48)
  Downloading pytz-2024.2-py2.py3-none-any.whl.metadata (22 kB)
Collecting PyYAML>=6.0.0 (from vminfo-parser==0.1.0.dev48)
  Using cached PyYAML-6.0.2-cp312-cp312-macosx_11_0_arm64.whl.metadata (2.1 kB)
Collecting six>=1.16.0 (from vminfo-parser==0.1.0.dev48)
  Using cached six-1.16.0-py2.py3-none-any.whl.metadata (1.8 kB)
Collecting tzdata>=2024.1 (from vminfo-parser==0.1.0.dev48)
  Using cached tzdata-2024.1-py2.py3-none-any.whl.metadata (1.4 kB)
Using cached contourpy-1.3.0-cp312-cp312-macosx_11_0_arm64.whl (251 kB)
Using cached cycler-0.12.1-py3-none-any.whl (8.3 kB)
Using cached et_xmlfile-1.1.0-py3-none-any.whl (4.7 kB)
Using cached fonttools-4.53.1-cp312-cp312-macosx_11_0_arm64.whl (2.2 MB)
Using cached kiwisolver-1.4.7-cp312-cp312-macosx_11_0_arm64.whl (63 kB)
Using cached matplotlib-3.9.2-cp312-cp312-macosx_11_0_arm64.whl (7.8 MB)
Using cached numpy-2.1.1-cp312-cp312-macosx_14_0_arm64.whl (5.1 MB)
Using cached openpyxl-3.1.5-py2.py3-none-any.whl (250 kB)
Using cached packaging-24.1-py3-none-any.whl (53 kB)
Using cached pandas-2.2.2-cp312-cp312-macosx_11_0_arm64.whl (11.3 MB)
Using cached pillow-10.4.0-cp312-cp312-macosx_11_0_arm64.whl (3.4 MB)
Using cached pyparsing-3.1.4-py3-none-any.whl (104 kB)
Using cached PyQt5-5.15.11-cp38-abi3-macosx_11_0_arm64.whl (6.6 MB)
Downloading PyQt5_Qt5-5.15.15-py3-none-macosx_11_0_arm64.whl (37.0 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 37.0/37.0 MB 43.8 MB/s eta 0:00:00
Using cached PyQt5_sip-12.15.0-cp312-cp312-macosx_10_9_universal2.whl (124 kB)
Using cached python_dateutil-2.9.0.post0-py2.py3-none-any.whl (229 kB)
Using cached python_magic-0.4.27-py2.py3-none-any.whl (13 kB)
Downloading pytz-2024.2-py2.py3-none-any.whl (508 kB)
Using cached PyYAML-6.0.2-cp312-cp312-macosx_11_0_arm64.whl (173 kB)
Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Using cached tzdata-2024.1-py2.py3-none-any.whl (345 kB)
Building wheels for collected packages: vminfo-parser
  Building wheel for vminfo-parser (pyproject.toml) ... done
  Created wheel for vminfo-parser: filename=vminfo_parser-0.1.0.dev48-py3-none-any.whl size=12726 sha256=f05f78650208ce8b111171a154271a0e3382386020b0c8fb31a7ab5db5bbf7fb
  Stored in directory: /private/var/folders/19/3zqrltys2d1byd2f900qr40w0000gn/T/pip-ephem-wheel-cache-ll33ce5h/wheels/5d/85/4e/9c79e8f683fbf9f181ff195025991f8ea37da6370574c8525f
Successfully built vminfo-parser
Installing collected packages: pytz, PyQt5-Qt5, tzdata, six, PyYAML, python-magic, PyQt5-sip, pyparsing, pillow, packaging, numpy, kiwisolver, fonttools, et-xmlfile, cycler, python-dateutil, PyQt5, openpyxl, contourpy, pandas, matplotlib, vminfo-parser
Successfully installed PyQt5-5.15.11 PyQt5-Qt5-5.15.15 PyQt5-sip-12.15.0 PyYAML-6.0.2 contourpy-1.3.0 cycler-0.12.1 et-xmlfile-1.1.0 fonttools-4.53.1 kiwisolver-1.4.7 matplotlib-3.9.2 numpy-2.1.1 openpyxl-3.1.5 packaging-24.1 pandas-2.2.2 pillow-10.4.0 pyparsing-3.1.4 python-dateutil-2.9.0.post0 python-magic-0.4.27 pytz-2024.2 six-1.16.0 tzdata-2024.1 vminfo-parser-0.1.0.dev48                                                                                   /11.6s
jaross@jaross1-mac ➜  testparse  venv:(.venv) vminfo-parser --help
usage: vminfo-parser [-h] (--file FILE | --yaml YAML) [--sort-by-env [SORT_BY_ENV]] [--prod-env-labels [PROD_ENV_LABELS]] [--generate-graphs] [--get-disk-space-ranges] [--show-disk-space-by-os] [--breakdown-by-terabyte]
                     [--over-under-tb] [--output-os-by-version] [--get-os-counts] [--os-name OS_NAME] [--minimum-count MINIMUM_COUNT] [--get-supported-os] [--get-unsupported-os]

Process VM CSV file

options:
  -h, --help            show this help message and exit
  --file FILE           The file to parse
  --yaml YAML           Path to YAML configuration file
  --sort-by-env [SORT_BY_ENV]
                        Sort disk by environment. Use "all" to get combine count, "both" to show both non-prod and prod, or specify one.
  --prod-env-labels [PROD_ENV_LABELS]
                        The values in your data that represent prod environments. This is used to generate prod and non-prod stats. Passed as CSV i.e. --prod-env-labels 'baker,dte'
  --generate-graphs     Choose whether or not to output visual graphs. If this option is not set, a text table will be outputted to the terminal
  --get-disk-space-ranges
                        This flag will get disk space ranges regardless of OS. Can be combine with --prod-env-labels and --sort-by-env to target a specific environment
  --show-disk-space-by-os
                        Show disk space by OS
  --breakdown-by-terabyte
                        Breaks disk space down into 0-2TB, 2-9TB and 9TB+ instead of the default categories
  --over-under-tb       A simple break down of machines under 1TB and those over 1TB
  --output-os-by-version
                        Output OS by version
  --get-os-counts       Generate a report that counts the inventory broken down by OS
  --os-name OS_NAME     The name of the Operating System to produce a report about
  --minimum-count MINIMUM_COUNT
                        Anything below this number will be excluded from the results
  --get-supported-os    Display a graph of the supported operating systems for OpenShift Virt
  --get-unsupported-os  Display a graph of the unsupported operating systems for OpenShift Virt
```

**Comment by csfreak** [comment]
blocked by #9

File: README.md [hunk 1]
```diff
@@ -5,3 +5,73 @@
...
+
+```
+git clone https://github.com/rhtools/vminfo-parser.git
+
+cd vminfo-parser
+pip3 install requirements.txt
+python3 -m vminfo_parser --help
+```
+
+## Usage
```
Comment (stratus-ss) [inline README.md:31]: Usage should also include a copy of the YAML or a reference to the files should it not?

**Review by stratus-ss — APPROVED** [review]
lgtm

**Pull request merged**